project/Dependencies.scala

```diff
@@ -137,6 +141,8 @@ object Dependencies {
     akkaTestkit,
     sprayTestkit,
     mockserverNetty,
-    mockito
+    mockito,
+    workbenchUtil,
```

Comment:
Could maybe omit these from `rawlsCoreDependencies` because it's already including `modelDependencies ++ googleDependencies ++ metricsDependencies`, all of which contain `workbenchUtil[Test]`